lib/data/subdivisions/VN.yaml

```diff
@@ -1,194 +1,194 @@
-    names: Quang Ngai
-- - '13'
-  - name: Quang Ninh
-    names: Quang Ninh
-- - '25'
-  - name: Quang Tri
-    names: Quang Tri
-- - '52'
-  - name: Soc Trang
-    names: Soc Trang
-- - '05'
-  - name: Son La
-    names: Son La
-- - '37'
-  - name: Tay Ninh
-    names: Tay Ninh
-- - '20'
-  - name: Thai Binh
-    names: Thai Binh
-- - '69'
-  - name: Thai Nguyen
-    names:
+'44':
+  name: An Giang
+  names: An Giang
+'43':
+  name: Ba Ria - Vung Tau
+  names: Ba Ria - Vung Tau
+'53':
+  name: Bac Can
+  names: Bac Can
+'54':
+  name: Bac Giang
+  names: Bac Giang
+'55':
+  name: Bac Lieu
+  names: Bac Lieu
+'56':
+  name: Bac Ninh
+  names: Bac Ninh
+'50':
+  name: Ben Tre
+  names: Ben Tre
+'31':
+  name: Binh Dinh
+  names: Binh Dinh
+'57':
+  name: Binh Duong
+  names: Binh Duong
+'58':
+  name: Binh Phuoc
+  names: Binh Phuoc
+'40':
+  name: Binh Thuan
+  names: Binh Thuan
+'59':
+  name: Ca Mau
+  names: Ca Mau
+'48':
+  name: Can Tho
+  names: Can Tho
+'04':
+  name: Cao Bang
+  names: Cao Bang
+'60':
+  name: Da Nang, thanh pho
+  names: Da Nang, thanh pho
+'33':
+  name: Dac Lac
+  names: Dac Lac
+'72':
+  name: Dak Nong
+  names: Dak Nong
+'71':
+  name: Dien Bien
+  names: Dien Bien
+'39':
+  name: Dong Nai
+  names: Dong Nai
+'45':
+  name: Dong Thap
+  names: Dong Thap
+'30':
+  name: Gia Lai
+  names: Gia Lai
+'03':
+  name: Ha Giang
+  names: Ha Giang
+'63':
+  name: Ha Nam
+  names: Ha Nam
+'64':
+  name: Ha Noi, thu do
+  names: Ha Noi, thu do
+'15':
+  name: Ha Tay
+  names: Ha Tay
+'23':
+  name: Ha Tinh
+  names: Ha Tinh
+'61':
+  name: Hai Duong
+  names: Hai Duong
+'62':
+  name: Hai Phong, thanh pho
+  names: Hai Phong, thanh pho
+'73':
+  name: Hau Giang
+  names: Hau Giang
+'65':
+  name: Ho Chi Minh, thanh pho [Sai Gon]
+  names: Ho Chi Minh, thanh pho [Sai Gon]
+'14':
+  name: Hoa Binh
+  names: Hoa Binh
+'66':
+  name: Hung Yen
+  names: Hung Yen
+'34':
+  name: Khanh Hoa
+  names: Khanh Hoa
+'47':
+  name: Kien Giang
+  names: Kien Giang
+'28':
+  name: Kon Tum
+  names: Kon Tum
+'01':
+  name: Lai Chau
+  names: Lai Chau
+'35':
+  name: Lam Dong
+  names: Lam Dong
+09
+  name: Lang Son
+  names: Lang Son
+'02':
+  name: Lao Cai
+  names: Lao Cai
+'41':
+  name: Long An
+  names: Long An
+'67':
+  name: Nam Dinh
+  names: Nam Dinh
+'22':
+  name: Nghe An
+  names: Nghe An
+'18':
+  name: Ninh Binh
+  names: Ninh Binh
+'36':
+  name: Ninh Thuan
+  names: Ninh Thuan
+'68':
+  name: Phu Tho
+  names: Phu Tho
+'32':
+  name: Phu Yen
+  names: Phu Yen
+'24':
+  name: Quang Binh
+  names: Quang Binh
+'27':
+  name: Quang Nam
+  names: Quang Nam
+'29':
+  name: Quang Ngai
+  names: Quang Ngai
+'13':
+  name: Quang Ninh
+  names: Quang Ninh
+'25':
+  name: Quang Tri
+  names: Quang Tri
+'52':
+  name: Soc Trang
+  names: Soc Trang
+'05':
+  name: Son La
+  names: Son La
+'37':
+  name: Tay Ninh
+  names: Tay Ninh
+'20':
+  name: Thai Binh
+  names: Thai Binh
+'69':
+  name: Thai Nguyen
+  names:
     - Central Highlands
```

Comment:
I think this is list syntax. I'm not sure why this needs to be a list if it has only one element, but I see other subdivision files that do the same.